shell/platform/common/text_editing_delta_unittests.cc

```diff
@@ -0,0 +1,37 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+//
+#include "flutter/shell/platform/common/text_editing_delta.h"
+
+#include "gtest/gtest.h"
+
+namespace flutter {
+
+TEST(TextEditingDeltaTest, TestTextEditingDeltaConstructor) {
+  // Here we are simulating inserting an "o" at the end of "hell".
+  std::string oldText = "hell";
+  std::string replacementText = "hello";
+  flutter::TextRange range = flutter::TextRange(0, 4);
```

Comment:
`flutter::` prefix isn't necessary here since you're already in the `flutter` namespace.

Just:

```
TextRange range(0, 4);
```